tests/lib/config/config-rule.js

```diff
@@ -304,9 +304,9 @@ describe("ConfigRule", () => {
         it("should allow to ignore deprecated rules", () => {
             const expectedRules = Array.from(builtInRules.entries())
                     .filter(([, rule]) => {
-                        const ruleSchema = (typeof rule === "function") ? rule.schema : rule.meta.schema;
+                        const isDeprecated = (typeof rule === "function") ? rule.deprecated : rule.meta.deprecated;
 
-                        return !ruleSchema.deprecated;
+                        return !isDeprecated;
```

Comment:
It might be a good idea to test for specific deprecated rules to make sure they're not on the list, to avoid issues like the `schema` mistake from earlier.